src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_getprojectservices.yaml

```diff
@@ -0,0 +1,36 @@
+post:
+  summary:  List all services related to a package.
+  description: |
+    List all services defined in the project spaces defined for this package.
```

Comment:
```suggestion
    List all services defined in the project space for this package.
```

Reply:
or maybe this describes it a little better? https://github.com/openSUSE/open-build-service/blob/fa5742de6c10e9ac12bc2ef1e3603f0229faa3c9/src/backend/bs_srcserver#L904

Reply:
I took the line of the old documentation, which states: "getprojectservices: list all service defined in project spaces defined for this package."

I also found tests for this endpoint: https://github.com/openSUSE/open-build-service/blob/8d1d9658185e39400b47718cb8c3bfdb6e528e08/src/api/test/functional/source_services_test.rb#L36-L57

In this case I prefer to follow the initial idea of "projects spaces". Is there any reason to switch to "project space" I'm missing?

Reply:
Anyway, it makes sense to remove `defined`.

Reply:
I removed the last `defined`.